packages/shortcuts-extension/src/components/ShortcutItem.tsx

```diff
@@ -376,12 +357,12 @@ export class ShortcutItem extends React.Component<
   getShortCutForDisplayOnly(key: string): JSX.Element[] {
     return this.props.shortcut.keys[key].map(
       (keyBinding: string, index: number) => (
-        <div className={ShortcutKeysContainerStyle} key={index}>
-          <div className={ShortcutKeysStyle} id={'shortcut-keys'}>
+        <div className="jp-Shortcuts-ShortcutKeysContainer" key={index}>
+          <div className="jp-Shortcuts-ShortcutKeys" id={'shortcut-keys'}>
```

Comment:
```suggestion
          <div className="jp-Shortcuts-ShortcutKeys">
```



I know this was not introduced in this PR but `shortcut-keys` identifier gets incorrectly re-used for each row of the table breaking HTML validity. Is it used elsewhere, or can we drop it?

Reply:
This extension is full of that kind of usage :sad:

That one at least does not seem to be reuse like that one :scream: 

https://github.com/jupyterlab/jupyterlab/blob/5582852e4af32269e2cd7f9ac01bce696b7713ea/packages/shortcuts-extension/src/components/ShortcutInput.tsx#L413-L414